src/components/home/home.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+
+import YTSDescription from 'components/yts_description';
+import PopularDownloads from 'containers/popular_downloads';
+import LatestTorrents from 'containers/latest_torrents';
+import { PageLoader } from 'components/loader';
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { pageLoader: false };
+  }
+
+  componentWillMount() {
+    this.setState({ pageLoader: true });
+  }
+
+  componentDidMount() {
+    this.setState({ pageLoader: false });
+  }
+  
+  render() {
+
+    return (
+        <div>
+          {this.state.pageLoader ? <PageLoader state={this.state.pageLoader} /> : null }
```

Comment:
Not sure about the PageLoader functionality here. It will just switch ON and OFF, without waiting for any actual data to come.

Reply:
I added a PageLoader to home page in case the data updates while we are on the other page of our application. In that case, if we choose to return to the home page, the loader will be activated while the new data is set up for home page.